repl/src/main/scala/com/cloudera/livy/repl/Session.scala

```diff
@@ -168,4 +195,21 @@ class Session(interpreter: Interpreter, stateChangedCallback: SessionState => Un
 
     compact(render(resultInJson))
   }
+
+  private def setJobGroup(statementId: Int): String = {
+    val cmd = Kind(interpreter.kind) match {
+      case Spark() | PySpark() | PySpark3() =>
+        s"""sc.setJobGroup("$statementId", "Job group for statement $statementId")"""
```

Comment:
From my test scala 2.11 repl will still occupy one placeholder:

```
scala> sc.setJobGroup("dfsa", "fds")

scala> 1
res1: Int = 1
```

As you mentioned 2nd statement "1" should be "res0", but it is "res1" as I tested.